source/_integrations/samsungtv.markdown

```diff
@@ -456,3 +467,9 @@ Some televisions from the H and J series use an encrypted protocol and require m
 The default setting on newer televisions is to ask for permission on every connection attempt.
 To avoid this behavior, please ensure that you adjust this to `First time only` in the `Device connection manager > Access notification` settings of your television.
 It is also recommended to cleanup the previous attempts in `Device connection manager > Device list`
+
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required.
```

Comment:
I'm not sure if we really need to add, that the following instructions isn't missing anything :grimacing:

Reply:
I think it makes sense to explicit says that there is no extra steps needed (one can think those are just not documented)

Reply:
yeah ... I'm fine with both :)